src/openklant/components/klantinteracties/api/serializers/klantcontacten.py

```diff
@@ -537,3 +537,59 @@ def create(self, validated_data):
         )
 
         return super().create(validated_data)
+
+
+class BetrokkeneConvenienceSerializer(BetrokkeneSerializer):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.fields["had_klantcontact"].read_only = True
+
+
+class OnderwerpobjectConvenienceSerializer(OnderwerpobjectSerializer):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.fields["klantcontact"].read_only = True
+
+
+class KlantContactBetrokkeneOnderwerpObjectSerializer(serializers.Serializer):
+    klantcontact = KlantcontactSerializer()
+    betrokkene = BetrokkeneConvenienceSerializer(required=False)
+    onderwerpobject = OnderwerpobjectConvenienceSerializer(required=False)
+
+    @transaction.atomic
+    def create(self, validated_data):
+        """
+        Create the objects and use the original serializers to ensure all the correct
+        fields show up in the response
+        """
+        klantcontact_data = validated_data["klantcontact"]
```

Comment:
Because it is required on the serializer, you will get a validation error:

```
{
  "type": "http://localhost:8000/ref/fouten/ValidationError/",
  "code": "invalid",
  "title": "Invalid input.",
  "status": 400,
  "detail": "",
  "instance": "urn:uuid:5f54a6d6-5327-4ad5-a831-27b7be3c1035",
  "invalidParams": [
    {
      "name": "klantcontact",
      "code": "required",
      "reason": "Dit veld is vereist."
    }
  ]
}
```